frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -612,6 +612,7 @@ func toDispatchState(ctx context.Context, dt []byte, opt ConvertOpt) (*dispatchS
 			llbCaps:           opt.LLBCaps,
 			sourceMap:         opt.SourceMap,
 			lintWarn:          opt.Warn,
+			argCmdVars:        make(map[string]struct{}),
```

Comment:
I don't fully understand why this new propery is needed, and also why it is part of "opt", not "dispatchState".

`dispatchState.buildargs` has same values I think